pkg/kapp/clusterapply/delete_change.go

```diff
@@ -32,10 +33,40 @@ type DeleteChange struct {
 	identifiedResources ctlres.IdentifiedResources
 }
 
+type uniqueResourceRef struct {
```

Comment:
How about we change this to "inoperableResourceRef" as we are loosely matching it without being strict about versions now? It is also not being used anywhere else.
(Others might have other ideas as well)

Reply:
Agree with the suggestion and updated the same. thanks